torch/csrc/jit/python/init.cpp

```diff
@@ -752,10 +753,31 @@ void initJITBindings(PyObject* module) {
       .def(
           "_jit_set_bailout_depth",
           [](size_t depth) {
+            TORCH_WARN("Use _jit_set_fusion_strategy, bailout depth is deprecated. Setting to (Dynamic, ", depth, ")");
             size_t old_depth = getBailoutDepth();
-            getBailoutDepth() = depth;
+            FusionStrategy strat = {{FusionBehavior::DYNAMIC, depth}};
```

Comment:
this will be changing the default behaviour where `_jit_set_bailout_depth` will only set `STATIC` strategies. Are we okay with this?

Reply:
sure let me update